packages/design-tokens/src/tokens/base/colors/light.json

```diff
@@ -32,7 +32,7 @@
             "value": "#8c959f"
           },
           "5": {
-            "value": "#6e7781"
+            "value": "#6a727c"
```

Comment:
@danielguillan qq. Is this the same value as what exists in Primer Primitives color scales?

Here's a scenario I'm thinking about...

Every now and then, we'll want to 'sync' our scales with upstream Primitives for shared hues (like gray). Right now it's not clear that there are overrides in play. 

Can we either:

- Reflect this change upstream
- Make note that this is an override, so future upgraders know to restore this color value. Might also help to explain why we changed it.